src/transport/HighPressureGasTransport.cpp

```diff
@@ -327,167 +323,579 @@ double HighPressureGasTransport::viscosity()
+    double Cv_mix = m_thermo->cv_mole(); // Units are J/kmol/K
+
+    // This result is in units of W/m/K
+    double thermal_conductivity = high_pressure_thermal_conductivity(tKelvin, T_star,
+                                                                     params.MW_mix, density,
+                                                                     Cv_mix, params.Vc_mix,
+                                                                     params.Tc_mix, params.sigma_mix,
+                                                                     params.acentric_factor_mix,
+                                                                     params.mu_r_mix, params.kappa_mix);
+
+    // Return the thermal conductivity in W/m/K
+    return thermal_conductivity;
+}
+
+double ChungHighPressureGasTransport::high_pressure_thermal_conductivity(double T, double T_star,
+                                                                        double MW, double rho,
+                                                                        double Cv, double Vc,
+                                                                        double Tc, double sigma,
+                                                                        double acentric_factor,
+                                                                        double mu_r, double kappa)
+{
+    // Calculate the low-pressure viscosity using the Chung method.
+    // This method returns viscosity in micropoise, but the thermal
+    // conductivity model needs the low-pressure viscosity to be in units of Pa*s.
+    double micropoise_to_pascals_second = 1e-7;
+    double viscosity = micropoise_to_pascals_second*low_pressure_viscosity(T, T_star, MW,
+                                                                           acentric_factor, mu_r,
+                                                                           sigma, kappa);
+
+    double M_prime = MW / 1000.0; // Convert kg/kmol to kg/mol
+
+    // Definition of tabulated coefficients for the Chung method, as shown in
+    // Table 10-3 on page 10.23.
+    vector<double> a = {2.44166, -5.0924e-1, 6.6107, 1.4543e1, 7.9274e-1, -5.8634, 9.1089e1};
+    vector<double> b = {7.4824e-1, -1.5094, 5.6207, -8.9139, 8.2019e-1, 1.2801e1, 1.2811e2};
+    vector<double> c = {-9.1858e-1, -4.9991e1, 6.4760e1, -5.6379, -6.9369e-1, 9.5893, -5.4217e1};
+    vector<double> d ={1.2172e2, 6.9983e1, 2.7039e1, 7.4344e1, 6.3173, 6.5529e1, 5.2381e2};
+
+    // This is slightly pedantic, but this is done to have the naming convention in the
+    // equations used match the variable names in the code. This is equation 10-5.9.
+    double B_1, B_2, B_3, B_4, B_5, B_6, B_7;
+    B_1 = a[0] + b[0]*acentric_factor + c[0]*pow(mu_r, 4.0) + d[0]*kappa;
+    B_2 = a[1] + b[1]*acentric_factor + c[1]*pow(mu_r, 4.0) + d[1]*kappa;
+    B_3 = a[2] + b[2]*acentric_factor + c[2]*pow(mu_r, 4.0) + d[2]*kappa;
```

Comment:
Generally, put the type declaration with the first assignment.
```suggestion
    double B_1 = a[0] + b[0]*acentric_factor + c[0]*pow(mu_r, 4.0) + d[0]*kappa;
    double B_2 = a[1] + b[1]*acentric_factor + c[1]*pow(mu_r, 4.0) + d[1]*kappa;
    double B_3 = a[2] + b[2]*acentric_factor + c[2]*pow(mu_r, 4.0) + d[2]*kappa;
```
etc.